src/components/ScreenContainer.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+const ScreenContainer = ({ children }) => {
+    return (
+        <View style={styles.ScreenContainer}>
+            { children }
+        </View>
+    )
```

Comment:
The CodeClimate tells them that you can try refactoring here 😊 

![image](https://user-images.githubusercontent.com/69128578/114465255-7c7a4480-9bac-11eb-8373-a7b36f24ce93.png)